config/flatcar/v1_0/translate.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2020 Red Hat, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.)
+
+package v1_0
+
+import (
+	"github.com/coreos/butane/config/common"
+	cutil "github.com/coreos/butane/config/util"
+	"github.com/coreos/butane/translate"
+
+	"github.com/coreos/ignition/v2/config/v3_3/types"
+	"github.com/coreos/vcontext/path"
+	"github.com/coreos/vcontext/report"
+)
+
+// ToIgn3_3Unvalidated translates the config to an Ignition config.  It also
+// returns the set of translations it did so paths in the resultant config
+// can be tracked back to their source in the source config.  No config
+// validation is performed on input or output.
+func (c Config) ToIgn3_3Unvalidated(options common.TranslateOptions) (types.Config, translate.TranslationSet, report.Report) {
+	ret, ts, r := c.Config.ToIgn3_3Unvalidated(options)
+	if r.IsFatal() {
+		return types.Config{}, translate.TranslationSet{}, r
+	}
+
+	for _, luks := range ret.Storage.Luks {
+		if luks.Clevis.IsPresent() {
+			r.AddOnError(path.New("json", "storage", "luks", "clevis"), common.ErrClevisSupport)
```

Comment:
This path is incorrect; it should be:

```suggestion
			r.AddOnError(path.New("json", "storage", "luks", i, "clevis"), common.ErrClevisSupport)
```

Also, it'd be good to add a test for this error, similar to: https://github.com/coreos/butane/blob/30a013ed4b934cd9a0cbbe534bd02066d93bfe4d/config/openshift/v4_11_exp/translate_test.go#L360-L478.